Dockerfile

```diff
@@ -4,7 +4,9 @@ WORKDIR $GOPATH
 
 # Download all the dependencies
 # https://stackoverflow.com/questions/28031603/what-do-three-dots-mean-in-go-command-line-invocations
-RUN go get -d -v  github.com/gorilla/mux
+RUN go get -d -v github.com/gorilla/mux
```

Comment:
You don't need to install dep here since you install it in the Makefile. Please make a choice between this two methods